src/tests/ftest/deployment/critical_integration.py

```diff
@@ -13,15 +13,97 @@
 from exception_utils import CommandFailure
 
 
-class CriticalIntegration(TestWithServers):
+"""
+    TO-DO
+    Provision all daos nodes using provisioning tool provided by HPCM
+"""
+
+class CriticalIntegrationWithoutServers(TestWithServers):
     """Test Class Description: Verify the basic integration of
                                the server nodes with available
                                framework and amongst themselves.
     :avocado: recursive
     """
 
-    #TO-DO
-    #Provision all daos nodes using provisioning tool provided by HPCM
+    def __init__(self, *args, **kwargs):
+        """Initialize the CriticalIntegrationWithoutServers class."""
+        super().__init__(*args, **kwargs)
+        self.start_agents_once = False
+        self.start_servers_once = False
+        self.setup_start_servers = False
+        self.setup_start_agents = False
+
+    def test_passwdlessssh_versioncheck(self):
+        # pylint: disable=protected-access
+        """
+        Test Description: Verify passwordless ssh amongst the server
+                          server nodes available and verify all server
+                          and client nodes have same daos versions.
+        :avocado: tags=all,deployment,full_regression
+        :avocado: tags=hw,large
+        :avocado: tags=criticalintegration,passwdlessssh_versioncheck
+        """
+
+        check_remote_root_access = self.params.get("check_remote_root_access", "/run/*")
+        daos_server_version_list = []
+        dmg_version_list = []
+        failed_nodes = []
+        for host in self.hostlist_servers:
+            daos_server_cmd = ("ssh -oNumberOfPasswordPrompts=0 {}"
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string